dcc-portal-ui/app/scripts/projects/js/project.js

```diff
@@ -290,4 +306,16 @@
 
     refresh();
   });
+
+  module.controller('CGPNodeModalCtl', function(
```

Comment:
this controller manages the modal, passes a centralised path to the files, and a URL for the policies.
should make things easier to change in the future.